cg/apps/cgstats/crud/create.py

```diff
@@ -127,6 +130,33 @@ def create_unaligned(
     return unaligned
 
 
+def create_dragen_unaligned(
+    manager: StatsAPI, demux_sample: DragenDemuxSample, sample_id: int, demux_id: int
+) -> stats_models.Unaligned:
+    unaligned: stats_models.Unaligned = manager.Unaligned()
```

Comment:
I don't know the reason, I based this off of what Måns did for bcl2fastq. I think he basically ported what was in `cgstats` into `cg`. Do you think I should add a pydantic model?

Reply:
:+1:

Reply:
Yes @barrystokman you are correct, that is just ported code as the PR would get to big if it would include total refactor as well. I suggest to do that in another PR